src/components/enterprise-page/data/hooks.js

```diff
@@ -5,13 +5,18 @@ import {
 import { logError, logInfo } from '@edx/frontend-platform/logging';
 import { camelCaseObject } from '@edx/frontend-platform/utils';
 
-import colors from '../../../colors.scss';
 import {
   fetchEnterpriseCustomerConfigForSlug,
   updateUserActiveEnterprise,
   fetchEnterpriseLearnerData,
 } from './service';
 
+const colors = {
+  primary: getComputedStyle(document.documentElement).getPropertyValue('--pgn-color-primary'),
+  info100: getComputedStyle(document.documentElement).getPropertyValue('--pgn-color-info-100'),
+  info500: getComputedStyle(document.documentElement).getPropertyValue('--pgn-color-info-500'),
+};
+
 export const defaultPrimaryColor = colors?.primary;
 export const defaultSecondaryColor = colors?.info100;
 export const defaultTertiaryColor = colors?.info500;
```

Comment:
nit: are the `?.` necessary in the above lines given we're defining `colors` as an object here?